src/gerbil/compiler/driver.ss

```diff
@@ -139,8 +139,13 @@ namespace: gxc
       (write `(define builtin-modules
                 (append (quote ,builtin-modules) libgerbil-builtin-modules)))
       (write `(define (gerbil-main)
-                (gerbil-runtime-init! builtin-modules)
-                (apply ,mod-main (cdr (command-line)))))
+                (with-catch
```

Comment:
no, it wont execute if there is no exceptio. Use with-protect.

Reply:
Name might be off, the unwind-protect fun basically.

Reply:
Trying again with `dynamic-wind`...